### ISSUE TITLE
Failed test 'Warning issued and alternative suggested'

### ISSUE THREAD

**Issue by eserte** [dallaylaen/perl-mvc-neaf#3]
On some of my smokers the t/911-Apache2-0.30.t test fails:
```
#   Failed test 'Warning issued and alternative suggested'
#   at t/911-Apache2-0.30.t line 53.
# found warning: WWW::Form::UrlEncoded::XS 0.25 is require. fallback to PP version at /usr/perl5.18.4p/lib/site_perl/5.18.4/amd64-freebsd/WWW/Form/UrlEncoded.pm line 17.
# found carped warning: MVC::Neaf::Request::Apache2 is DEPRECATED and will be REMOVED in 0.30. Use Plack::Handler::Apache2 instead. at t/911-Apache2-0.30.t line 52.
# expected to find warning: (?^:DEPRECATED.*Plack::Handler::Apache)
# Looks like you failed 1 test of 7.
t/911-Apache2-0.30.t .................. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/7 subtests 
```

This happens if XS and PP version of WWW::Form::UrlEncoded do not match. See also https://github.com/kazeburo/WWW-Form-UrlEncoded/issues/3

**Comment by dallaylaen** [comment]
Thank you! Is there a link to the full run? Anyway I'll try to fix it...

**Comment by eserte** [comment]
All FAILs by me (SREZIC) are probably caused by this problem: http://fast-matrix.cpantesters.org/?dist=MVC-Neaf%200.2602;reports=1

**Comment by dallaylaen** [comment]
http://www.cpantesters.org/cpan/report/cceffade-f02f-11e8-a27d-80ede8504111

This has been fixed by 0.2603. Thanks again for you contribution!
